pkg/controllers/volume-mgmt/handler.go

```diff
@@ -757,3 +757,32 @@ func (c *CStorVolumeController) reconcileVersion(cv *apis.CStorVolume) (*apis.CS
 	}
 	return cv, nil
 }
+
+// markCVStatusToOffline will fetch all the CV resources present
+// in etcd and mark it them CV.Status to Offline
+func (c *CStorVolumeController) markCVStatusToOffline() {
+	openebsNameSpace := os.Getenv("OPENEBS_NAMESPACE")
+	cvList, err := c.clientset.CstorV1().CStorVolumes(openebsNameSpace).List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		klog.Errorf("failed to fetch CV list, error: %v", err)
+		return
+	}
+	for i := range cvList.Items {
+		// Checking if Valid Cstor
+		cv := cvList.Items[i]
+		if IsValidCStorVolumeMgmt(&cv) {
+			cv.Status.Phase = apis.CVStatusOffline
+			cv.Status.ReplicaStatuses = nil
+			
+			// updating cv status to offline
+			_, err = c.clientset.CstorV1().CStorVolumes(cv.Namespace).Update(context.TODO(), &cv, metav1.UpdateOptions{})
+			if err != nil {
+				klog.Errorf("failed to update CV: %s status to %s", cv.Name, apis.CVStatusOffline)
+				continue
```

Comment:
Here also, we found the CV that required to update and processing other CV's will not helpful so we can return
```suggestion
				return
```